webrender/src/device/gfx/device.rs

```diff
@@ -172,15 +179,15 @@ pub struct Device<B: hal::Backend> {
     pub depth_format: hal::format::Format,
     pub queue_group_family: QueueFamilyId,
     pub queue_group_queues: Vec<B::CommandQueue>,
-    command_pools: SmallVec<[CommandPool<B>; 1]>,
+    command_pools: SmallVec<[CommandPool<B>; FRAME_COUNT_MAILBOX]>,
```

Comment:
in cases where you know it's a *hard* bound, we should use `ArrayVec` instead of `SmallVec`